client/a8c-for-agencies/sections/sites/sites-dashboard/sites-dataviews-style.scss

```diff
@@ -0,0 +1,80 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.sites-dataviews__site {
+	display: flex;
+	flex-direction: row;
+
+	.button {
+		padding: 0;
+	}
+}
+
+.sites-dataviews__site-name {
+	display: inline-block;
+	text-align: left;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	white-space: nowrap;
+	width: 180px;
+	font-weight: 500;
+	font-size: rem(14px);
+}
+
+.sites-dataviews__site-url {
+	text-overflow: ellipsis;
+	overflow: hidden;
+	white-space: nowrap;
+	font-size: rem(12px);
```

Comment:
Somehow, sometimes this font size is used (12px) and sometimes this one is used (14px), which is the correct one?

https://github.com/Automattic/wp-calypso/blob/d1021f7b28ca9ece95f52015b80548942ea5165e/client/sites-dashboard/components/sites-site-url.ts#L10

Reply:
I guess for Dotcom, it will be 14px. At A4A, we don't have the `SiteUrl` component.